xpdan/data_reduction.py

```diff
@@ -152,10 +152,17 @@ def _prepare_header_list(headers):
     return header_list
 
 
-def _load_config():
-    with open(
-            os.path.join(an_glbl.config_base, an_glbl.calib_config_name)) as f:
-        config_dict = yaml.load(f)
+def _load_config(header):
```

Comment:
I think you also need to change line [247](https://github.com/xpdAcq/xpdAn/pull/32/files#diff-79db28d59f034ae0a4b4bc04efb5c88eR247) to take the header